pages/index.ejs

```diff
@@ -1,11 +1,45 @@
-header: <fix>CMintS</fix> starter
+title: Privacy Manager
+description: Privacy Manager website
 ---
 
-<h2>{homepage[Page heading] Homepage}</h2>
-
-<p>{homepage-p[Paragraph in "Homepage" section] This is the homepage that is using <fix>EJS</fix>.}</p>
-
-<a href="/page2">{homepage-p-link[Link to the second page in "Homepage" section] Go to Page2}</a>
+<section class="col2 container">
+  <div>
+    <h2>{privacy-manager-header[Header of the section] Privacy Manager}</h2>
+    <ul>
+      <li>{privacy-manager-li1[Privacy Manager section bullet point] Privacy Management}</li>
+      <li>{privacy-manager-li2[Privacy Manager section bullet point] <fix>Cookie</fix> Management}</li>
+      <li>{privacy-manager-li3[Privacy Manager section bullet point] Browsing data deletion on browser start}</li>
+      <li>{privacy-manager-li4[Privacy Manager section bullet point] Network monitoring}</li>
+    </ul>
+    <a id="install-button" href="https://chrome.google.com/webstore/detail/privacy-manager/giccehglhacakcfemddmfhdkahamfcmd" target=_blank>Get Privacy Manager for Chrome</a>
+  </div>
+  <iframe class="tablet-hide end" id="promo-video" src="https://www.youtube.com/embed/iaKweivaOhI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</section>
+<section class="bg-secondary">
+  <div class="col3 center container cards">
+    <h3 class="cell1-3">{what-you-get[Privacy Manager section h3 title] What <em>You Get</em>}</h3>
+    <div>
+      <img src="/images/trancparency.svg" alt="{what-u-get-trasparency-alt[Privacy manager transparency image alt text] Magnifier and newspaper}" />
+      <h4>{what-u-get-trasparency-heading[Privacy Manager section h4 title] Transparency}</h4>
+      <p>{what-u-get-trasparency-p[Privacy manager transparency text] Something about Transparency, Something about Transparency Something about Transparency}</p>
+    </div>
+    <div>
+      <img src="/images/keep-under-control.svg" alt="{what-u-get-control-alt[Privacy manager content control image alt text] Content control}" />
+      <h4>{what-u-get-control-heading[Privacy Manager section h4 title] Keep Under Control}</h4>
+      <p>{what-u-get-control-p[Privacy manager text about keeping under control] Something about Control Something about Control Something about Control}</p>
+    </div>
+    <div>
+      <img src="/images/stay-secure.svg" alt="{what-u-get-secure-alt[Privacy manager stay secure image alt text] Folder with a shield}" />
+      <h4>{what-u-get-secure-heading[Privacy Manager section h4 title] Stay Secure}</h4>
+      <p>{what-u-get-secure-p[Privacy manager text about staying secure] Something about Security Something about Security Something about Security}</p>
+    </div>
+  </div>
+</section>
+<section id="media" class="col4 center container">
+  <h3 class="cell1-4">{what-media-say[Privacy Manager section h3 title] What Media <em>Says</em>}</h3>
```

Comment:
Suggestion: we can remove h3 from description, because there's no other heading in the section.

Reply:
Sure, try using similar pattern as was suggested here -> https://github.com/Privacy-Managers/privacy-manager-website/pull/6#discussion_r494186301